### ISSUE TITLE
[Feature Request] Quark  like "Easy Transfer" buttons and features in sophisticated backpacks and storage

### ISSUE THREAD

**Issue by alexnmt** [P3pp3rF1y/SophisticatedBackpacks#1029]
**Describe the bug**
The Insert and Extract buttons from Quark's Easy Tranfer feature don't show up in either backpacks or storage. The key commands to insert and extract items also don't work. This means we can't match items to store in the target inventory by shift clicking and hitting insert or extract and makes it harder to quickly manage inventories. 

**Screenshots**
<img width="473" alt="Screenshot 2024-04-09 at 22 19 34" src="https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/33011297/31c09e1d-2da5-4781-b571-f5afd66ea83a">
Vanilla inventory with Quark installed - Insert/extract buttons in bottom right under top inventory
<img width="671" alt="Screenshot 2024-04-09 at 22 19 20" src="https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/33011297/752b0c21-b7e7-41e9-aade-bea15f3eefe0">
Sophisticated Storage with Quark installed - no insert/extract buttons under inventory. 
<img width="614" alt="Screenshot 2024-04-09 at 22 19 44" src="https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/33011297/2ba25224-9b2e-41c3-adfe-34519e7daaff">
Sophisticated Storage with Quark installed

**Versions**
Sophisticated Backpacks 1.20.1-3.20.5.1039
Sophisticated Core 1.20.1-0.6.18.597
Sophisticated Storage-1.20.1-0.10.20.778
(Quark 4.0-439)

**Comment by P3pp3rF1y** [comment]
Yes they don't show up because they are disabled in Quark by default for custom inventories and would actually require reimplementation of this quark feature in sophisticated because default quark implementaiton would cause item loss / dupes with some of the sophisticated features.

I want to take a look at having this implementation again as I had in 1.18, but it will likely be just there by default even without Quark because anyway the whole thing had to be implemented on sophisticated side and the only thing that was used from Quark in 1.18 was the button look.

So I am going to change this to feature request for Quark like transfer in sophisticated.

**Comment by alexnmt** [comment]
oh that's interesting that in the old versions it wasn't actually quark doing it! I have been so used to the feature from playing so much VH in 1.18, so thanks for looking into re-implementing it in newer versions. 

The advanced deposit upgrade with the match inventory contents is nice to help with transferring inventories, at least when going directly into the storage, so that helps in the mean time.

**Comment by P3pp3rF1y** [comment]
I have added these buttons some time ago, just didn't close this issue so if you use any release in the past 2 months or so you will see them there.
